src/Utilities/AddGoal.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+
+export class AddGoal extends Component {
```

Comment:
I think this would be better off as a function rather than a component. 
But we can keep it like this for now if you want